From fb6701ffeab9104d4bdec3e905ae9b1978de6d9f Mon Sep 17 00:00:00 2001
From: Christian Kerl <christian.kerl@in.tum.de>
Date: Mon, 10 Feb 2014 16:16:15 +0100
Subject: [PATCH] increased max iso buffer length for linux usbfs

---
 libusb/os/linux_usbfs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libusb/os/linux_usbfs.h b/libusb/os/linux_usbfs.h
index 1f5b191..29836a0 100644
--- a/libusb/os/linux_usbfs.h
+++ b/libusb/os/linux_usbfs.h
@@ -81,7 +81,7 @@ struct usbfs_iso_packet_desc {
 	unsigned int status;
 };
 
-#define MAX_ISO_BUFFER_LENGTH		32768
+#define MAX_ISO_BUFFER_LENGTH		49152
 #define MAX_BULK_BUFFER_LENGTH		16384
 #define MAX_CTRL_BUFFER_LENGTH		4096
 
-- 
1.7.9.5

